index.js

```diff
@@ -3983,6 +3983,26 @@
   S.splitOn =
   def('splitOn', {}, [$.String, $.String, $.Array($.String)], splitOn);
 
+  //# splitOnRegex :: RegExp -> String -> Array String
+  //.
+  //. Returns the substrings of its second argument separated by evaluated
+  //. matches of its first argument.
```

Comment:
Thank you for mentioning capturing groups, @cust0dian. They do make this function more difficult to accurately describe. Perhaps the best option is to provide an incomplete one-sentence description and a link to [MDN][1]. What do you think?


[1]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split